Makefile

```diff
@@ -11,14 +11,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-START_PROMETHEUS=true
-
 all: test-integration
 
 test-integration:
-ifndef CLUSTER_NAME
-  $(error CLUSTER_NAME is undefined)
-endif
-	go test ./... -args -integration -cluster-name=$(CLUSTER_NAME) -start-prometheus=$(START_PROMETHEUS)
+	go test ./... -args -integration -cluster-location=$(GCP_REGION) -cluster-name=$(KUBE_CLUSTER) -namespace-name=$(KUBE_NAMESPACE) -project-id=$(GCP_PROJECT)
```

Comment:
As we have multiple required parameters to run e2e test, add a section under README on how you run it helps. for example:
```
GCP_REGION=us-east-1b KUBE_CLUSTER=e2e-cluster KUBE_NAMESPACE=test-namespace GCP_PROJECT=test-project make test-integration
```

Reply:
Good idea.